chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/QuestionableUrlReportGenerator.java

```diff
@@ -71,6 +71,8 @@ public void execute(JobExecutionContext jobContext) throws JobExecutionException
         SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
         LOGGER.info("********* Starting the Questionable URL Report Generator job. *********");
         activeStatuses.add(CertificationStatusType.Active);
+        activeStatuses.add(CertificationStatusType.SuspendedByAcb);
```

Comment:
It's not clear to me if or how the code change will not apply the filter to ADMIN/ONC users getting the report.

Reply:
This variable is only used in the `isUrlRelatedToAcbs` and `isUrlRelatedToActiveListing` methods, which are only called from the `filterUrls` method, which only calls them if the `if (isAcbSpecific(jobContext)) {` check returns "true". Basically, the filtering happens iff it's the ONC-ACB job